include/verilogAST.hpp

```diff
@@ -487,15 +520,13 @@ class ModuleInstantiation : public StructuralStatement {
 
   std::string instance_name;
 
-  // map from instance port names to connection expression
   // NOTE: anonymous style of module connections is not supported
-  std::map<std::string, std::unique_ptr<Expression>> connections;
+  std::unique_ptr<Connections> connections;
```

Comment:
don't need this to be dynamically allocated but fine either way.

Reply:
If I don't use a dynamic allocation I get an issue with a deleted copy constructor
```
/Users/leonardtruong/repos/verilogAST-cpp/include/verilogAST.hpp:533:9: err
or:
      call to deleted constructor of 'verilogAST::Connections'
        connections(std::move(connections)){};
        ^           ~~~~~~~~~~~~~~~~~~~~~~
/Users/leonardtruong/repos/verilogAST-cpp/include/verilogAST.hpp:490:3: not
e:
      'Connections' has been explicitly marked deleted here
  Connections(const Connections&) = delete;
```

I would need to add a copy constructor that clones the pointers